# Array names should be plural nouns (array-names-plural)

Variables that are initialized with array literals
or properties in objects that have array literal value
should have name that is plural noun, starting with lowercase letter.

## Rule Details



Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
